creator-node/src/services/stateMachineManager/stateMonitoring/StateMonitoringQueue.js

```diff
@@ -173,55 +178,47 @@ class StateMonitoringQueue {
     })
   }
 
-  /**
-   * Registers the logic that gets executed to process each new job from the queue.
-   * @param {Object} queue the StateMonitoringQueue to consume jobs from
-   */
-  registerQueueJobProcessor(queue) {
-    // Initialize queue job processor (aka consumer)
-    queue.process(1 /** concurrency */, async (job) => {
-      const {
-        id: jobId,
-        data: {
-          lastProcessedUserId,
-          discoveryNodeEndpoint,
-          moduloBase,
-          currentModuloSlice
-        }
-      } = job
-
-      try {
-        this.log(`New job details: jobId=${jobId}, job=${JSON.stringify(job)}`)
-      } catch (e) {
-        this.logError(`Failed to log details for jobId=${jobId}: ${e}`)
+  async processJob(job) {
+    const {
+      id: jobId,
+      data: {
+        lastProcessedUserId,
+        discoveryNodeEndpoint,
+        moduloBase,
+        currentModuloSlice
       }
+    } = job
 
-      // Default results of this job will be passed to the next job, so default to failure
-      let result = {
+    try {
+      this.log(`New job details: jobId=${jobId}, job=${JSON.stringify(job)}`)
+    } catch (e) {
+      this.logError(`Failed to log details for jobId=${jobId}: ${e}`)
+    }
```

Comment:
it's a remnant of the past when we weren't seeing any new job logs and wondering if it's because some error happened -- like maybe stringifying the job led to an error because the JSON object was circular. since we know the job data is good now, I'd say it's safe to remove that try. will do that in another PR 👍